rasa/core/policies/policy.py

```diff
@@ -120,12 +126,17 @@ def __init__(
         self.__featurizer = self._create_featurizer(featurizer)
         self.priority = priority
         self.finetune_mode = should_finetune
+        self._rule_only_data = {}
 
     @property
     def featurizer(self):
         """Returns the policy's featurizer."""
         return self.__featurizer
 
+    def set_shared_policy_states(self, **kwargs: Any) -> None:
```

Comment:
can we please mention this change in the changelog?

Reply:
do you mean the fact that we added it? it's not really a change, no?

Reply:
it's a change to a public interface of Rasa Open Source so users might also want to implement it for their own `Policy`s